src/layout/PageWithNavigation.tsx

```diff
@@ -41,14 +52,26 @@ export const PageWithNavigation = (props: Props) => {
   };
 
   return (
-    <Container style={{paddingBottom:'6rem'}}>
+    <Container style={{ paddingBottom: '6rem' }}>
       <Box mb={3}>
-        <ChooseMishnaBar 
-        allChapterAllowed={allChapterAllowed}
-        onNavigationSelected={navigationSelectedHandler}
-         />
+        <ChooseMishnaBar
+          allChapterAllowed={allChapterAllowed}
+          onNavigationSelected={navigationSelectedHandler}
+        />
+      </Box>
+      <Box
```

Comment:
this surrounding <Box> div affects the layout of the page (the child page is a container)
It needs to be separate from it